src/services/BackendService.ts

```diff
@@ -1,9 +1,14 @@
-import {action, observable, makeObservable, runInAction} from "mobx";
-import {CARTA} from "carta-protobuf";
-import {Observable, Observer, Subject, throwError} from "rxjs";
-import {AppStore, PreferenceStore, RegionStore} from "stores";
-import {ApiService} from "./ApiService";
-import {mapToObject} from "utilities";
+import { action, observable, makeObservable, runInAction } from "mobx";
+
```

Comment:
where do these new lines come from?

Reply:
It comes from TSlint. Do we need this lint?

Reply:
which tslint rule in particular? I don't mean the spaces between `{` and `action`, those are fine. I mean the blank lines between imports.

Reply:
get the problem and correct it.